logos/src/logos/classification/laura_embedding_classifier.py

```
@@ -0,0 +1,80 @@
+
+    def remove_model(self, model_id: str):
+        if model_id in self.model_db:
+            del self.model_db[model_id]
+            self.save_db()
+
+    def classify_prompt(self, prompt: str, top_k: int = 1):
+        """Returns top-k most similar model IDs for a given prompt."""
+        if not self.model_db:
+            return []
+        query_emb = self.encode_text(prompt, prefix="query:")
+        model_ids = list(i for i in self.model_db.keys() if i in self.allowed or not self.allowed)
```

Ruff (0.11.9)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)